### ISSUE TITLE
Streets of Arkham support (1.4)

### ISSUE THREAD

**Issue by NPBruce** [NPBruce/valkyrie#654]

**Comment by NPBruce** [comment]
Atlas images have changed, so Valkyrie is bugged until MoM 1.4 support is released.

**Comment by NPBruce** [comment]
I notice that chemical symbols are a new option for master mind.  We can add that.

**Comment by NPBruce** [comment]
ImagePuzzle1_background image has been removed.  This will be broken if used by old scenarios (not a lot that can be done).

**Comment by NPBruce** [comment]
Anyone able to tell me what the keywords are for the new items?

**Comment by NPBruce** [comment]
I also need photos of both sides of the tokens to work out who is who.

**Comment by Hobbeslionheart** [comment]
I will get my copy tonight or tomorrow morning and will get help you then.

**Comment by RhOmicron** [comment]
Do I understand correctly. You just need the text on the new items and the front/back picture of the new NPCs?

**Comment by Hobbeslionheart** [comment]
Needs to match up name with the icons so as long you flipped them all in the same places, that works.

He just needed the type (evidence, relic, firearm, etc), so I think we're good there. Surprised no new common item unless relics are common items.

**Comment by RhOmicron** [comment]
Fixing it up now

**Comment by NPBruce** [comment]
Horror and Awareness numbers are also required.

**Comment by RhOmicron** [comment]
Monster Front
https://imgur.com/a/r1WIw
Monster Back
https://imgur.com/a/1i8OV

**Comment by NPBruce** [comment]
Great, thanks for your help.

**Comment by NPBruce** [comment]
Does anyone know what the deal is with the Elixir?  It is listed as a unique item, but the border is a different colour and it isn't in RhOmicron's list.

**Comment by NPBruce** [comment]
btw... "Azure Flame" as a spell, and there is a Road to Legend quest full of "Green Flame".  I think someone at FFG likes Acquisitions Inc.

**Comment by NPBruce** [comment]
Another question, which tiles back on to which other tiles?:


TILE_DINER_MAD25,Diner 
TILE_EXHIBIT_ENTRANCE_MAD25,Exhibit Entrance 
TILE_YARD_1_MAD25,Yard 1 
TILE_MAIN_EXHIBIT_MAD25,Main Exhibit 
TILE_LAB_MAD25,Lab 
TILE_BANDSTAND_MAD25,Bandstand 
TILE_LOBBY_MAD25,Lobby 
TILE_EXHIBIT_3_MAD25,Exhibit 3 
TILE_CLASSROOM_1_MAD25,Classroom 1 
TILE_LIBRARY_MAD25,Library 
TILE_GENERAL_SHOP_MAD25,General Shop 
TILE_STUDIO_MAD25,Studio 
TILE_SEWER_1_MAD25,Sewer 1 
TILE_LIBRARY_ARCHIVES_MAD25,Library Archives 
TILE_PRINTING_PRESS_MAD25,Printing Press 
TILE_EXHIBIT_2_MAD25,Exhibit 2 
TILE_STREET_CORNER_MAD25,Street Corner 
TILE_RESEARCH_LIBRARY_MAD25,Research Library 
TILE_SEWER_2_MAD25,Sewer 2 
TILE_CHAPEL_MAD25,Chapel 
TILE_SPEAKEASY_MAD25,Speakeasy 
TILE_PORTRAIT_HALL_MAD25,Portrait Hall 
TILE_MAGIC_SHOP_MAD25,Magic Shop 
TILE_OFFICE_MAD25,Office 
TILE_YARD_2_MAD25,Yard 2 
TILE_BEDROOM_MAD25,Bedroom 
TILE_ALLEY_MAD25,Alley 
TILE_EXHIBIT_1_MAD25,Exhibit 1 
TILE_CURIOSITY_SHOP_MAD25,Curiosity Shop 
TILE_HALL_STAIRS_MAD25,Hall Stairs 
TILE_WAREHOUSE_MAD25,Warehouse 
TILE_CLASSROOM_2_MAD25,Classroom 2 
TILE_CLASSROOM_3_MAD25,Classroom 3 
TILE_LOUNGE_MAD25,Lounge 

**Comment by NPBruce** [comment]
Yet another question!

Do the sewers count as inside or outside? (text colour)

**Comment by NPBruce** [comment]
Outstanding work:
Is the elixir a unique item?  How is it used?
Are the sewer tiles inside or outside (only used as an editor aid)?
Which sides are the reverse of which other sides (only used as an editor aid)?
Add MAD25 Mythos events (can be done later)
Add all monsters activations (currently only one per new monster, this should be done before release)
Add monster health values (This should be done before release)

**Comment by RhOmicron** [comment]
Elixir - Spell "At the start of your turn you may improve 1 skill of your choice(new tokens). If you do, flip this card.

Sewer tiles - Inside (brown)

Tiles
Stairs / Curiosity Shop
Printing Press / Exhibit 2
Speakeasy / Portrait Hall
Street Corner / Research Library
Sewer 1 / Library Archives
Sewer 2 / Chapel
Shop Office, Magic Shop / Office
Exhibit 1 / Alley
Classroom 2 / Warehouse
Lounge / Classroom 3
Bedroom / Yard 2
- Medium Tiles -
Lobby / Exhibit 3
Yard 1 / Main Exhibit
Classroom 1 / Library
Bandstand / Lab
Shop Storage, General Shop / Studio, Studio Storage
- Large Tile -
Kitchen, Diner / Exhibit Entrance

**Comment by NPBruce** [comment]
Great thanks!  Now I just have to work out how to deal with a unique spell?

**Comment by NPBruce** [comment]
1.6.0c has all but monster activations, monster health and extra mythos events.

**Comment by NicolaZee** [comment]
Outstanding work on getting SoA implemented.

There is minor issue with the latest release 1.6.1 re SoA.
If the new SoA character Tommy Muldoon is selected then his Unique Item Becky (rifle) is not automatically included in the list of Starting Items.

I don't know if this has been fixed in development or if this should be raised as a separate issue.

**Comment by redwolf2** [comment]
@NicolaZee https://github.com/NPBruce/valkyrie/issues/673
